### ISSUE TITLE
MXNet binary 404 Error

### ISSUE THREAD

**Issue by mendl7** [dmlc/mxnet#5895]
Tried to do a windows RStudio install following the instructions 
"install.packages("drat", repos="https://cran.rstudio.com")
drat:::addRepo("dmlc")
install.packages("mxnet")"

I got the below error.
## Error Message:
"Installing package into ‘/rpackages/lsjohn1’
(as ‘lib’ is unspecified)
downloaded 0 bytes

Warning in install.packages :
  URL 'http://dmlc.ml/drat/src/contrib/mxnet_0.9.4.tar.gz': status was '404 Not Found'
Error in download.file(url, destfile, method, mode = "wb", ...) : 
  cannot download all files
Warning in install.packages :
  download of package ‘mxnet’ failed"

R `sessionInfo()`:
sessionInfo()
R version 3.3.1 (2016-06-21)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 14.04.5 LTS

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C               LC_TIME=en_US.UTF-8       
 [4] LC_COLLATE=en_US.UTF-8     LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                  LC_ADDRESS=C              
[10] LC_TELEPHONE=C             LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] BiocInstaller_1.24.0

loaded via a namespace (and not attached):
[1] drat_0.1.2  tools_3.3.1

## Minimum reproducible example
"install.packages("drat", repos="https://cran.rstudio.com")
drat:::addRepo("dmlc")
install.packages("mxnet")"

**Comment by thirdwing** [comment]
From the information you provided,

```
Platform: x86_64-pc-linux-gnu (64-bit)
```
I am afraid you are on Linux not Windows.

The drat installation is for Windows and OSX.

**Comment by mendl7** [comment]
ah, the joys of cloud-based computing. Thanks for catching that.

**Comment by coforfe** [comment]
Hello,

I have just updated to R 3.4.0 in my Mac (macSierra) machine and when trying to install `mxnet` I get this error (attached is my `version`):

```r
> install.packages("mxnet", repos="http://dmlc.ml/drat/", type="mac.binary")
Warning in install.packages :
  cannot open URL 'http://dmlc.ml/drat/bin/macosx/contrib/3.4/PACKAGES.rds': HTTP status was '404 Not Found'
Warning in install.packages :
  cannot open URL 'http://dmlc.ml/drat/bin/macosx/contrib/3.4/PACKAGES.gz': HTTP status was '404 Not Found'
Warning in install.packages :
  cannot open URL 'http://dmlc.ml/drat/bin/macosx/contrib/3.4/PACKAGES': HTTP status was '404 Not Found'
Warning in install.packages :
  unable to access index for repository http://dmlc.ml/drat/bin/macosx/contrib/3.4:
  no fue posible abrir la URL 'http://dmlc.ml/drat/bin/macosx/contrib/3.4/PACKAGES'

   package ‘mxnet’ is available as a source package but not as a binary

Warning in install.packages :
  package ‘mxnet’ is not available (as a binary package for R version 3.4.0)
```
And the `version` of my system is this:

```r
> version
               _                           
platform       x86_64-apple-darwin15.6.0   
arch           x86_64                      
os             darwin15.6.0                
system         x86_64, darwin15.6.0        
status                                     
major          3                           
minor          4.0                         
year           2017                        
month          04                          
day            21                          
svn rev        72570                       
language       R                           
version.string R version 3.4.0 (2017-04-21)
nickname       You Stupid Darkness         
```

Thanks,
Carlos.
